internal/pkg/agent/install/install.go

```diff
@@ -229,6 +225,103 @@ func Install(cfgFile, topPath string, unprivileged bool, log *logp.Logger, pt *p
 	return ownership, nil
 }
 
+func copyFiles(streams *cli.IOStreams, pathMappings []map[string]string, dir string, topPath string) error {
```

Comment:
I would like to see more comments sprinkled throughout this function, focussing more on why we're doing the various copying and link fixing steps here.

Reply:
done in 3781d6f702bc010d806c1e44e0a865c234206f20